### ISSUE TITLE
Bugs and enhancements to fix up clone-tutorial.sh

### ISSUE THREAD

**Issue by ybyzek** [confluentinc/kafka-tutorials#466]
I ran the `clone-tutorial.sh` utility and hit some issues

1. There is no sample properties file provided.  Suggestion: add comments into `clone-tutorial.sh` that show example properties file
2. Checklist file is not in the path specified in the output. 

Output:

```
 "Addling link    <li><a href="garbage/filtering.html">MEANINGFUL LINK TEXT HERE</a></li>
 to /Users/yeva/git/kafka-tutorials/_includes/tutorials/garbage/_checklist.txt
```

Actual file location

```
/Users/yeva/git/kafka-tutorials/_includes/tutorials/garbage/garbage_checklist.txt
```

3. The folder under` _includes/tutorial/newKT` is incorrect — it has an extra path to the old folder

```
[14:45:44] ~/git/kafka-tutorials/_includes/tutorials/garbage(GH-446) ✗: ls
total 4
drwxr-xr-x  4 yeva staff  128 Jul 17 14:44 .
drwxr-xr-x 35 yeva staff 1120 Jul 17 14:44 ..
drwxr-xr-x  4 yeva staff  128 Jul 17 14:44 filtering
-rw-r--r--  1 yeva staff  609 Jul 17 14:44 garbage_checklist.txt
```

**Comment by bbejeck** [comment]
Also, check that in all cases the  script updates `.semaphore/semaphore.yml` file with an entry for the new tutorial.

**Comment by bbejeck** [comment]
resolved via #544
